### PR TITLE
feat: remove tvl cap from omnipool

### PR DESCRIPTION
**Pull request by enthusiastmartin**
This PR
- removes TVL cap from omnipool
- removes stable asset config parameter
- removes initialize_pool extrinsic as it is no longer needed

### CODE REVIEW

File: pallets/omnipool/src/lib.rs [hunk 1]
```diff
@@ -566,6 +443,13 @@ pub mod pallet {
 
 			ensure!(initial_price > FixedU128::zero(), Error::<T>::InvalidInitialAssetPrice);
 
+			// ensure collection is created, we can simply ignore the error if it was already created.
+			let _ = T::NFTHandler::create_collection(
+				&T::NFTCollectionId::get(),
+				&Self::protocol_account(),
+				&Self::protocol_account(),
+			);
+
```
Comment (enthusiastmartin) [inline pallets/omnipool/src/lib.rs:452]: @mrq1911 i dont really like this here. 

I would remove it and let the creation be as prerequisite.

**Comment by Roznovjak** [comment]
Fix the tests please

File: runtime/hydradx/src/weights/omnipool.rs [hunk 2]
```diff
@@ -54,32 +54,6 @@ use pallet_omnipool::weights::WeightInfo;
 pub struct HydraWeight<T>(PhantomData<T>);
 
 impl<T: frame_system::Config> WeightInfo for HydraWeight<T> {
```
Comment (mrq1911) [inline runtime/hydradx/src/weights/omnipool.rs:56]: add token should be re-benched with initialization in mind

**Review by mrq1911 — APPROVED** [review]
fix tests and LGTM

**Pull request merged**